### ISSUE TITLE
Linux: support a few older glibc versions with AppImage distro

### ISSUE THREAD

**Issue by eoliphan** [warpdotdev/Warp#4393]
### Describe the solution you'd like?

Hi, this may or may not be worth it to you guys, I see that the .AppImage supports at least glibc 2.27.  But there are still, unfortunately, some common desktop envs still using slightly older versions.  I use a Mac for my day to day personal stuff, so i'm all good, but one of my clients (where i'm trying to demo warp) is all in on AWS Linux (plain, not Ubuntu) Workspaces.  Those are still based on AWS Linux 2, which only supports glibc 2.26, and AWS has yet to announce the upgrade to AWS Linux 2023, which is based on glibc 2.30+.  

### Is your feature request related to a problem? Please describe.

Unable to run the warp linux app image on AWS (plain) Linux Workspace due to this env only supporting glibc 2.26, the rpm distro is obviously even less lenient in this regard.

### How important is this feature to you?

3

### Warp Internal (ignore) - linear-label:39cc6478-1249-4ee7-950b-c428edfeecd1

None

**Comment by LFd3v** [comment]
Hi, there. First, I am not a Warp developer. Second, as far as I know, `glibc` would not be the only problem for a very old Linux distribution, but also that the graphics library being used (wgpu) and requires [Vulkan or OpenGL ES 3.0+](https://docs.warp.dev/getting-started/getting-started-with-warp#linux).

**Comment by vorporeal** [comment]
Yeah, we currently build on Ubuntu 20.04 LTS, but plan on moving to 18.04 LTS, which would hopefully further reduce glibc requirements.  No ETA yet, but I'll let you know here when that work is done!
